week-5_1

TatSPb opened the pull request:
добавлены сервисы для работы с изображениями
добавлены тесты
добавлен README-файл

TatSPb committed:
week-5_1
добавлены сервисы для работы с изображениями
добавлены тесты
добавлен README-файл

BizinMitya commented on src/main/java/ru/skypro/homework/controller/SearchController.java:
```diff
@@ -0,0 +1,27 @@
+    private final AdService adService;
+
+    @GetMapping
+    public ResponseEntity<?> searchAds(@PathVariable String req) throws UnauthorizedException {
```
Это эндпоинт не нужен, его нет в спецификации

TatSPb committed:
week-6
1) удалила эндпоинт и сервис searchAds согласно твоим комментариям, так как его нет в спецификации;
2) удалила лишние комментарии из проекта.

TatSPb committed:
Merge pull request #11 from TatSPb/week-6

week-6
